gentelella/settings.py

```diff
@@ -83,8 +83,8 @@
         # 'NAME': 'diaspora_validation',  # Completao
         'NAME': 'sigs_test_feature',
         # 'NAME': 'sigs_validation',
-        'USER': 'postgres',
-        'PASSWORD': 'postgres',
+        'USER': 'vitorribas',
+        'PASSWORD': 'vitorribas',
```

Comment:
We should maintain the default postgres user and password.

Reply:
Updated.